services/frontend/src/pages/HeroStatsPage.tsx

```
@@ -0,0 +1,114 @@
+
+  function SortIcon({ col }: { col: keyof HeroStatsResponse }) {
+    if (sortKey !== col) return <span className="sort-icon">↕</span>
+    return <span className="sort-icon active">{sortAsc ? '↑' : '↓'}</span>
+  }
+
+  return (
+    <div>
+      <div className="page-header">
+        <h1>Hero Stats</h1>
+        <div className="filters">
+          <label>Position:</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)